scripts/mk_usb_installer.sh

```diff
@@ -32,7 +32,8 @@ dev=$1
 
 # easy: leave this as is.
 # it will use the server run at the end of this script.
-preseed="url=$(hostname):8000"
+# preseed="url=$(hostname):8000"
+preseed="url=10.100.7.247:8000"
```

Comment:
oh yeah..  working around wonky network.  $(hostname) should work well some and maybe most  of the time ;)

ill fix